job-service/src/main/java/com/hpe/caf/services/job/api/DatabaseHelper.java

```diff
@@ -300,6 +346,59 @@ public boolean doesJobAlreadyExist(String jobId, int jobHash) throws Exception {
     }
 
     /**
+     * Returns TRUE if the specified job id is complete, otherwise FALSE.
+     */
+    public boolean isJobComplete(final String jobId) throws Exception
```

Comment:
Why is isJobComplete() needed?

Reply:
No longer needed, I have removed this from the latest commit